dashboard/test/fixtures/script.yml

```diff
@@ -88,6 +88,16 @@ minecraft:
   user_id:
   login_required: false
   properties: '{}'
+hero:
+  id: 72
+  name: hero
+  created_at: 2015-10-16 22:39:24.000000000 Z
+  updated_at: 2015-10-16 22:39:24.000000000 Z
+  wrapup_video_id:
+  hidden: true
+  user_id:
+  login_required: false
+  properties: '{}'
 frozen:
```

Comment:
@balderdash helped me figure out that the tests are likely failing to find the hero script which is causing the failures because the hero script isn't loaded in for tests. I added it here to the fixtures, which should fix the issue 🤞